types/serve-static/serve-static-tests.ts

```diff
@@ -13,7 +14,7 @@ app.use(serveStatic('/3', {
     lastModified: true,
     maxAge: 0,
     redirect: true,
-    setHeaders: function(res: express.Response, path: string, stat: any) {
+    setHeaders: function(res, path: string, stat: any) {
```

Comment:
(does work with `: express.Response` too ofc)

Reply:
Please add an `// $ExpectType`  on the next line to verify the contextual type is still `express.Response`.